README.md

```diff
@@ -7,341 +7,171 @@ This library provides debugging functionality to Ruby (MRI) 2.6 and later.
-=>#0    <main> at target.rb:5
-%self => main
-a => 1
-b => 2
-c => nil
-d => nil
-
-(rdbg) continue                        # Continue the execution
-[3, 11] in target.rb
-      3| a = 1
-      4| b = 2
-      5| binding.break
-      6| c = 3
-      7| d = 4
-=>    8| binding.break                 # Again the program stops here
-      9| p [a, b, c, d]
-     10|
-     11| __END__
-=>#0    <main> at target.rb:8
-
-(rdbg) info locals                     # And you can see the updated local variables
-=>#0    <main> at target.rb:8
-%self => main
-a => 1
-b => 2
-c => 3
-d => 4
-
-(rdbg) continue
-[1, 2, 3, 4]
-```
-
-### Invoke the program from the debugger as a traditional debuggers
-
-If you don't want to modify the source code, you can set breakpoints with a debug command `break` (`b` for short).
-Using `rdbg` command to launch the program without any modifications, you can run the program with the debugger.
-
-```shell
-$ cat target.rb                        # Sample program
-a = 1
-b = 2
-c = 3
-d = 4
-p [a, b, c, d]
-
-$ rdbg target.rb                       # run like `ruby target.rb`
-DEBUGGER: Session start (pid: 7656)
-[1, 7] in target.rb
-=>    1| a = 1
-      2| b = 2
-      3| c = 3
-      4| d = 4
-      5| p [a, b, c, d]
-      6|
-      7| __END__
-=>#0    <main> at target.rb:1
-
-(rdbg)
-```
-
-`rdbg` command suspends the program at the beginning of the given script (`target.rb` in this case) and you can use debug commands. `(rdbg)` is prompt. Let's set breakpoints on line 3 and line 5 with `break` command (`b` for short).
-
-```shell
-(rdbg) break 3                         # set breakpoint at line 3
-#0  BP - Line  /mnt/c/ko1/src/rb/ruby-debug/target.rb:3 (line)
+# Usage
 
-(rdbg) b 5                             # set breakpoint at line 5
-#1  BP - Line  /mnt/c/ko1/src/rb/ruby-debug/target.rb:5 (line)
+The debugger is designed to support a wide range of use cases, so you have many ways to use it.
 
-(rdbg) break                           # show all registered breakpoints
-#0  BP - Line  /mnt/c/ko1/src/rb/ruby-debug/target.rb:3 (line)
-#1  BP - Line  /mnt/c/ko1/src/rb/ruby-debug/target.rb:5 (line)
-```
+But essentially, it consists of 4 steps:
 
-You can see that two breakpoints are registered. Let's continue the program by `continue` command.
+1. [Activate the debugger in your program](#activate-the-debugger-in-your-program)
+1. Set breakpoints
+    - Through [`binding.break`](#the-bindingbreak-method)
+    - [Breakpoint commands](#breakpoint)
+1. Execute/continue your program and wait for it to hit the breakpoints
+1. Start debugging
+    - Here's the [full commands list](#console-commands)
+    - You can also type `help` or `help <command>` in the console to see commands
 
-```shell
-(rdbg) continue
-[1, 7] in target.rb
-      1| a = 1
-      2| b = 2
-=>    3| c = 3
-      4| d = 4
-      5| p [a, b, c, d]
-      6|
-      7| __END__
-=>#0    <main> at target.rb:3
+> **NOTE**
```

Comment:
I think you need lower case for it to work.

<img width="966" alt="Screen Shot 2022-07-07 at 9 47 27 AM" src="https://user-images.githubusercontent.com/15860699/177789337-aae4a877-0f87-42b7-9a51-73b354062941.png">

<img width="873" alt="Screen Shot 2022-07-07 at 9 47 42 AM" src="https://user-images.githubusercontent.com/15860699/177789405-d6402c3f-26a1-404e-979d-9f79b44f5835.png">

```suggestion
> **Note**
```